ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -947,7 +959,8 @@ func (r *RayClusterReconciler) createHeadPod(ctx context.Context, instance rayv1
 
 	// build the pod then create it
 	pod := r.buildHeadPod(ctx, instance)
-	if EnableBatchScheduler {
+	// this option is deprecated and will be removed soon
```

Comment:
I think the comment is meant for `EnableBatchScheduler`. Did you forget to remove it?

Reply:
Addressed.